docs/deployment.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deployment and CI/CD Process
+---
+
+# Deployment and CI/CD Process
+
+## Overview
+
+DEV relies on GitHub + Travis to continuous deploy to Heroku. If a Pull Request is merged without the "ci skip" in it's title, it will be automatically deployed to production once the test suite passes. The process currently takes about 20 minute to complete and will need a few addtional minute before the change live.
+
+## Travis steps
+
+The following steps can be explored in our [.travis.yml](https://github.com/thepracticaldev/dev.to/blob/master/.travis.yml) and [Procfile](https://github.com/thepracticaldev/dev.to/blob/master/Procfile)
+
+1. Travis runs the test portion of Rails code
+1. Travis runs the test portion of Preact code
```

Comment:
Do you want these all to be "1." or increment? Figured some might run in parallel but the final steps I know are synchronous.

Reply:
I think we could run these in parallel in theory

Reply:
The all "1"s is an acceptable Markdown Syntax. You can view [the file itself](https://github.com/thepracticaldev/dev.to/blob/848d3aedddf498ab36aed50f6500a96be8c746d0/docs/deployment.md) to see that the numbers are properly incremented.

Reply:
TIL